bedevere/bpo.py

```diff
@@ -21,7 +23,7 @@
 SKIP_ISSUE_LABEL = util.skip_label("issue")
 STATUS_CONTEXT = "bedevere/issue-number"
 # Try to keep descriptions at or below 50 characters, else GitHub's CSS will truncate it.
-_FAILURE_DESCRIPTION = 'No issue # in title or "skip issue" label found'
+_FAILURE_DESCRIPTION = 'Issue does not exit, No issue # in title or "skip issue" label found'
```

Comment:
This felt a little dirty, but I was not sure if we wanted to "yet another" edge case of failure description. Happy to refactor this if we want to be more specific of the reason for failure.

Reply:
Hmm, this might be too long as a status check message.
I'm thinking it can be a separate status check, maybe "bedevere/valid-issue"? @brettcannon, what do you think?

Reply:
I don't think we need another status check (we are already at 4). Is there any reason the message can't be dynamically specified like for the `NEWS.d` check? In this instance the current message is the one that shows up if anything is found, but if an issue number is discovered but found not to exist, a different message saying something like "issue #NNNN not found at bugs.python.org" or something would be set while still flagging a status check failure.

Make sense?

Reply:
Using a different message sounds good :)

Reply:
Corrections to message: /exit/exist/, /No/no/, /title/title,/.  But +1 to situation specific message.   This is not an exception raised in inner loops for control flow.  (The reason for some quick and vague exception messages.)